tests/testthat/test_fragments.R

```diff
@@ -34,14 +35,9 @@ test_that("calculateFragments", {
         z = 2,
         seq = c(rep(c("A", "AC"), 3),
                 rep(c("E", "CE"), 3)),
+        peptide = rep("ACE", 12),
         stringsAsFactors=FALSE)
-
-    expect_message(calculateFragments("PQR", verbose = TRUE),
-                   "Modifications used: C=57.02146")
-    expect_message(calculateFragments("PQR", modifications = NULL,
```

Comment:
Maybe we should add a unit test `expect_warning(calculateFragments("PQR", modifications = c(P=2)), "deprecated")`?

Reply:
Good point.